docs/plugins/inputs/kafka.asciidoc

```diff
@@ -3,23 +3,11 @@
 
 
 
-This input will read events from a Kafka topic. It uses the high level consumer API provided
-by Kafka to read messages from the broker. It also maintains the state of what has been
-consumed using Zookeeper. The default input codec is json.
-
-Here's a compatibility matrix that shows the Kafka client versions that are compatible with each combination
-of Logstash and the Kafka input plugin: 
-
-[options="header"]
-|==========================================================
-|Kafka Client Version |Logstash Version |Plugin Version |Security Features |Why?
-|0.8       |2.0.0 - 2.x.x   |<3.0.0 | |Legacy, 0.8 is still popular 
-|0.9       |2.0.0 - 2.3.x   | 3.x.x |Basic Auth, SSL |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
-|0.9       |2.4.0 - 5.0.x   | 4.x.x |Basic Auth, SSL |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
-|0.10      |2.4.0 - 5.0.x   | 5.x.x |Basic Auth, SSL |Not compatible with the 0.9 broker 
-|==========================================================
-
-NOTE: We recommended that you use matching Kafka client and broker versions. During upgrades, you should upgrade brokers before clients because brokers target backwards compatibility. For example, the 0.9 broker is compatible with both the 0.8 consumer and 0.9 consumer APIs, but not the other way around.
+This input will read events from a Kafka topic. 
+
+This plugin uses Kafka Client 0.9.0.1. For broker compatibility, see the official https://cwiki.apache.org/confluence/display/KAFKA/Compatibility+Matrix[Kafka compatibility reference].
```

Comment:
I see that the 2.4 branch [locks jruby-1.5.0](https://github.com/elastic/logstash/blob/2.4/Gemfile.jruby-1.9.lock#L337-L338) and that the [jruby-1.5.0 gem depends on kafka 0.8 jars](https://rubygems.org/gems/jruby-kafka/versions/1.5.0-java).

Reply:
Interesting, my prior research was wrong --

```
⓿ pork(~/build/logstash-2.4.1)
% find ./ -name '*kafka-client*'
./vendor/bundle/jruby/1.9/gems/jruby-kafka-1.5.0-java/lib/org/apache/kafka/kafka-clients
./vendor/bundle/jruby/1.9/gems/jruby-kafka-1.5.0-java/lib/org/apache/kafka/kafka-clients/0.8.2.2/kafka-clients-0.8.2.2.jar
```

Reply:
Fixed.